images/runtime/python/template.Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN ${IMAGES_DIR}/runtime/python/install-dependencies.sh
 RUN pip install --upgrade pip \
     && pip install gunicorn \
     && pip install debugpy \
+    && if [ "%PYTHON_MAJOR_VERSION%" = "3" ] && [ "%PYTHON_VERSION%" != "3.6" ]; then pip install viztracer \
+    && pip install vizplugins \
+    && pip install orjson ;fi \
```

Comment:
nit: May we update semicolon on line 44 to the following: `&& pip install orjson; fi \`

Reply:
Updated